src/style.css

```diff
@@ -1 +1,28 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(/src/images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 0;
+}
```

Comment:
The `:not(:first-child)` pseudo-class is unnecessary here because you've already set `margin-right` on `.stars__star`. If you want to remove the margin for the last child, you should use `:last-child` instead to override the right margin.